scripts/globals/msg.lua

```diff
@@ -311,3 +311,17 @@ tpz.msg.basic =
     TRUST_NO_CAST_TRUST    = 700,  -- You are unable to use Trust magic at this time.
     TRUST_NO_CALL_AE       = 717,  -- You cannot call forth alter egos here.
 }
+
+------------------------------------
+-- Message System
+------------------------------------
+
```

Comment:
since you renamed the paclket, maybe rename the table to match its new name?

Reply:
OH! Ima dunce there are yellow system3 type aren't they, i mistook this as an enum for the combatMessage packet you did.